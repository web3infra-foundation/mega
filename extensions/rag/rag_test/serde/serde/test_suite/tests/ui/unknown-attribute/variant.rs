use serde_derive::Serialize;

#[derive(Serialize)]
enum E {
    #[serde(abc = "xyz")]
    V,
}

fn main() {}
