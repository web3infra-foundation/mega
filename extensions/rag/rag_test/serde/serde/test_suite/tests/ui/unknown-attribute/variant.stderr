error: unknown serde variant attribute `abc`
 --> tests/ui/unknown-attribute/variant.rs:5:13
  |
5 |     #[serde(abc = "xyz")]
  |             ^^^
